### PR TITLE
Add translation strings to "columns" pattern 

### LINKED ISSUE #37: Add translation strings to "columns" pattern
Translation strings need to be added to the columns pattern. For example:

**Before**
`<li>Showcase your projects</li> `

**After**
`<li><?php echo esc_html__( 'Showcase your projects', 'twentytwentyfour' ); ?> </li>`

### PR DESCRIPTION
**Pull request by damonsharp**
This addresses issue #37 and adds translation functions to text in the columns pattern.

Normally at Kanopi we enforce certain standards and formatting, especially line lengths, so I broke some content for that reason. Can adjust if needed.

### CODE REVIEW

File: patterns/columns.php [hunk 1]
```diff
@@ -12,51 +12,64 @@
 <div class="wp-block-group alignfull has-base-background-color has-background" style="padding-top:7em;padding-right:7em;padding-bottom:7em;padding-left:7em"><!-- wp:columns {"align":"wide"} -->
 <div class="wp-block-columns alignwide"><!-- wp:column {"width":"33%","style":{"spacing":{"blockGap":"","padding":{"right":"12%"}}}} -->
 <div class="wp-block-column" style="padding-right:12%;flex-basis:33%"><!-- wp:heading {"level":3} -->
-<h3 class="wp-block-heading">We recognize the crucial role architecture plays in shaping a sustainable future.</h3>
+<h3 class="wp-block-heading">
+    <?php echo esc_html__( 'We recognize the crucial role architecture plays in shaping a sustainable future.', 'twentytwentyfour' ); ?>
```
Comment (kishanjasani) [inline patterns/columns.php:16]: Please use a single-function for echoing a single translation string.
```suggestion
    <?php esc_html_e( 'We recognize the crucial role architecture plays in shaping a sustainable future.', 'twentytwentyfour' ); ?>
```

File: patterns/columns.php [hunk 2]
```diff
@@ -12,51 +12,64 @@
...
 <!-- /wp:heading --></div>
 <!-- /wp:column -->
 
 <!-- wp:column {"style":{"spacing":{"blockGap":"5.6em"}}} -->
 <div class="wp-block-column"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap"}} -->
 <div class="wp-block-group"><!-- wp:group {"layout":{"type":"flex","orientation":"vertical","justifyContent":"left"}} -->
 <div class="wp-block-group"><!-- wp:heading {"level":4,"style":{"typography":{"lineHeight":"1.1","fontStyle":"normal","fontWeight":"700"}},"fontSize":"medium","fontFamily":"system-font"} -->
-<h4 class="wp-block-heading has-system-font-font-family has-medium-font-size" style="font-style:normal;font-weight:700;line-height:1.1">Consulting</h4>
+<h4 class="wp-block-heading has-system-font-font-family has-medium-font-size" style="font-style:normal;font-weight:700;line-height:1.1">
+    <?php echo esc_html__( 'Consulting', 'twentytwentyfour' ); ?>
```
Comment (kishanjasani) [inline patterns/columns.php:26]:
```suggestion
    <?php esc_html_e( 'Consulting', 'twentytwentyfour' ); ?>
```

File: patterns/columns.php [hunk 3]
```diff
@@ -12,51 +12,64 @@
...
+</h4>
 <!-- /wp:heading -->
 
 <!-- wp:paragraph -->
-<p>Our vision is to be at the forefront of architectural innovation, fostering a global community of architects and enthusiasts united by a passion for creating spaces.</p>
+<p><?php echo esc_html__( 'Our vision is to be at the forefront of architectural innovation, fostering a global community of architects and enthusiasts united by a passion for creating spaces.', 'twentytwentyfour' ); ?></p>
```
Comment (kishanjasani) [inline patterns/columns.php:31]:
```suggestion
<p><?php esc_html_e( 'Our vision is to be at the forefront of architectural innovation, fostering a global community of architects and enthusiasts united by a passion for creating spaces.', 'twentytwentyfour' ); ?></p>
```

File: patterns/columns.php [hunk 4]
```diff
@@ -12,51 +12,64 @@
...
 <!-- /wp:paragraph --></div>
 <!-- /wp:group -->
 
 <!-- wp:group {"layout":{"type":"flex","orientation":"vertical"}} -->
 <div class="wp-block-group"><!-- wp:heading {"level":4,"style":{"typography":{"lineHeight":"1.1","fontStyle":"normal","fontWeight":"700"}},"fontSize":"medium","fontFamily":"system-font"} -->
-<h4 class="wp-block-heading has-system-font-font-family has-medium-font-size" style="font-style:normal;font-weight:700;line-height:1.1">Project Management</h4>
+<h4 class="wp-block-heading has-system-font-font-family has-medium-font-size" style="font-style:normal;font-weight:700;line-height:1.1">
+    <?php echo esc_html__( 'Project Management', 'twentytwentyfour' ); ?>
```
Comment (kishanjasani) [inline patterns/columns.php:38]:
```suggestion
    <?php esc_html_e( 'Project Management', 'twentytwentyfour' ); ?>
```

**Comment by kishanjasani** [comment]
@damonsharp Could I request you to format the code? code spacing and indentation are not properly formatted.

Thanks

**Comment by carolinan** [comment]
HI @kishanjasani, Some contributors are first-time contributors to the WordPress project.
Can you provide more information about what you mean by "properly formatted"? 
-Explain briefly that WordPress has several different coding standards, where to find the information about the standards, and an example of what needs to be fixed?

**Comment by kishanjasani** [comment]
Thanks @carolinan. 

@damonsharp Coding standards and code formatting are essential aspects of maintaining a consistent and readable codebase. They ensure that our projects are not only functional but also comprehensible to all contributors, which greatly aids collaboration and future maintenance.

**Coding Standards:**
Coding standards are a set of guidelines that define how code should be written within a project. They encompass everything from naming conventions to indentation, from commenting practices to structuring files. These standards provide a unified approach that helps developers understand and work with each other's code seamlessly.

**Formatted Code:**
Formatted code follows the coding standards meticulously. It involves consistent indentation, proper spacing, and adherence to naming conventions. When code is well-formatted, it's not just syntactically correct but also easy to read and understand.

For your convenience, I recommend referring to the official WordPress Coding Standards documentation. This guide outlines the expected formatting practices, helping you differentiate between correct and incorrect methods. You can find the HTML indentation section here: [WordPress Coding Standards - HTML Indentation](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/html/#indentation)

By aligning your code with these standards, you contribute to the overall health and professionalism of the project. It also allows for smoother collaboration as other developers review and work with your code.

If you have any questions or need further assistance, please don't hesitate to reach out.

**Comment by damonsharp** [comment]
@carolinan / @kishanjasani -

Thanks. I'm aware of the PHPCS coding standards as I was part of the push to implement them at Kanopi. PHPStorm either reverted it before I committed or I have no idea what happened. Either way I didn't do a final check on the submission, so thanks for the kickback! Formatting changes pushed.

At Kanopi we also implement SonarQube standards for line length and other items, which this is not completely passing, but going to leave those out for now. If you need further adjustments though please let me know. Thanks.

**Comment by juanfra** [comment]
Thanks for working on this, Damon!

I am still seeing formatting issues with the proposed changes. An excellent way to see all the comments that the linting tool is returning is by running the following commands:

`composer install && composer run standards:check`

You can find more information about testing on the [project's readme](https://github.com/WordPress/twentytwentyfour#testing).

--

We also would need to replace the escaping functions to add context to the strings.

`echo esc_html_x( 'text', 'context', 'text-domain' )`

**Comment by MaggieCabrera** [comment]
Sorry I am late to this one. It seems that the strings have already been translated by another PR. I'm going to close this since I think rebasing this just to add formatting is going to be a bit of a pain, it's probably better if we open a new one for that.

**Comment by damonsharp** [comment]
It's okay. Been down for the count with Covid since the conference anyway and never had a moment to revisit. Glad it was handled elsewhere.

**Comment by MaggieCabrera** [comment]
> It's okay. Been down for the count with Covid since the conference anyway and never had a moment to revisit. Glad it was handled elsewhere.

Sorry to hear that! Hope you are feeling better

**Pull request closed without merging**